google/cloud/internal/log_impl_test.cc

```diff
@@ -61,6 +61,35 @@ TEST(CircularBufferBackend, Basic) {
   EXPECT_THAT(be->ExtractLines(), ElementsAre("msg 9", "msg 10"));
 }
 
+TEST(PerThreadCircularBufferBackend, Basic) {
+  auto be = std::make_shared<ScopedLog::Backend>();
+  PerThreadCircularBufferBackend buffer(3, Severity::GCP_LS_ERROR, be);
```

Comment:
Opt:  It took me a while to grok the significance of the `3` in the expectations (here, and in the `CircularBufferBackend` version).  Perhaps using a suitably-named variable could alleviate some of that?

Reply:
Done. Also add some comments and make the per-thread test more interesting.